emcc.py

```diff
@@ -395,6 +395,22 @@ def apply_settings(user_settings):
     if key == 'WASM_OBJECT_FILES':
       settings.LTO = 0 if value else 'full'
 
+  apply_min_browser_versions(user_settings)
+
+
+def default_min_browser_version(user_settings, browser, version):
+  default_setting(user_settings, f"MIN_{browser.upper()}_VERSION", version)
+
+
+# apply minimum browser version defaults based on user settings. if
+# a user requests a feature that we know is only supported in browsers
+# from a specific version and above, we can assume that browser version.
+def apply_min_browser_versions(user_settings):
+  if settings.WASM_BIGINT:
+    default_min_browser_version(user_settings, "Safari", 150000)
+    default_min_browser_version(user_settings, "Edge", 79)
+    default_min_browser_version(user_settings, "Firefox", 68)
```

Comment:
Please move that comment about chrome to here. Could be something like `The minimum required chrome version for WASM_BIGINT is 75, and we require at least that much already anyhow.` (phrased that way, it will remain valid if/when we raise the global minimum version from 75 some day)